AdyenTwint/AdyenTwint.docc/AdyenTwint.md

```diff
@@ -0,0 +1,114 @@
+```
+
+### Components
+Create a `TwintPaymentMethod` instance from your payment methods list.
+```
+let paymentMethod = paymentMethods.paymentMethod(ofType: TwintPaymentMethod.self)
+```
+
+Create the the `TwintComponent` with the `paymentMethod` and the `configuration`.
+```
+let component = TwintComponent(paymentMethod: paymentMethod, context: adyenContext, configuration: configuration)
+```
+
+Set your `AdyenSession` instance as the delegate to let it handle the flow.
+```
+component.delegate = self.adyenSession
+```
+
+Your app must configure a `callbackAppScheme` so that it is reopened once the payment is done.
+
+```
+componentConfig.actionComponent.twint = .init(callbackAppScheme: yourReturnURLScheme)
+```
+
+### Note:
+
+1. Your app should call the `RedirectComponent` in the `AppDelegate` by adding below method.
+
+```
+func application(_ app: UIApplication, open url: URL, options: [UIApplicationOpenURLOptionsKey: Any] = [:]) -> Bool {
+    RedirectComponent.applicationDidOpen(from: url)
+    return true
+}
+```
+
+2. Your app needs to register all apps that will be opened by your app.To do
+this, the following entries need to be added to the your App’s Info.plist file. If this is not done, then TWINT SDK does not find any installed TWINT app.
+
+```
+<key>LSApplicationQueriesSchemes</key>
+<array>
+<string>twint-issuer1</string>
+<string>twint-issuer2</string>
+<string>twint-issuer3</string>
+<string>twint-issuer4</string>
+<string>twint-issuer5</string>
+<string>twint-issuer6</string>
+<string>twint-issuer7</string>
+<string>twint-issuer8</string>
+<string>twint-issuer9</string>
+<string>twint-issuer10</string>
+<string>twint-issuer11</string>
+<string>twint-issuer12</string>
+<string>twint-issuer13</string>
+<string>twint-issuer14</string>
+<string>twint-issuer15</string>
+<string>twint-issuer16</string>
+<string>twint-issuer17</string>
+<string>twint-issuer18</string>
+<string>twint-issuer19</string>
+<string>twint-issuer20</string>
+<string>twint-issuer21</string>
+<string>twint-issuer22</string>
+<string>twint-issuer23</string>
+<string>twint-issuer24</string>
+<string>twint-issuer25</string>
+<string>twint-issuer26</string>
+<string>twint-issuer27</string>
+<string>twint-issuer28</string>
+<string>twint-issuer29</string>
+<string>twint-issuer30</string>
+<string>twint-issuer31</string>
+<string>twint-issuer32</string>
+<string>twint-issuer33</string>
+<string>twint-issuer34</string>
+<string>twint-issuer35</string>
+<string>twint-issuer36</string>
+<string>twint-issuer37</string>
+<string>twint-issuer38</string>
+<string>twint-issuer39</string>
+<string>twint-issuer40</string>
+<string>twint-issuer41</string>
+<string>twint-issuer42</string>
+<string>twint-issuer43</string>
+<string>twint-issuer44</string>
+<string>twint-issuer45</string>
+<string>twint-issuer46</string>
+<string>twint-issuer47</string>
+<string>twint-issuer48</string>
+<string>twint-issuer49</string>
+<string>twint-issuer50</string>
```

Comment:
This could be confusing in the end to any iOS developer who is aware of the limitation of 50 entries. Let's think of some advices we can provide merchant teams with? Like any idea if we can have a mapping of url scheme and a specific app provider? Like which of them to sacrifice?

Reply:
I will come back to you on this.